hassio/src/dashboard/hassio-update.ts

```diff
@@ -140,6 +160,23 @@ export class HassioUpdate extends LitElement {
           display: inline-block;
           margin-bottom: 32px;
         }
+        .icon {
+          --iron-icon-height: 48px;
+          --iron-icon-width: 48px;
+          float: right;
+          margin: 0 0 2px 10px;
+        }
+        .update-heading {
+          font-size: var(--paper-font-subhead_-_font-size);
+          font-weight: bold;
+          margin-bottom: 0.5em;
+        }
+        .warning {
+          color: var(--google-yellow-700);
```

Comment:
Let's use secondary text color for this, yellow on white is not so nice.

Reply:
Sounds good.